src/@next/pages/CreateJobAd/JobPostingDetailsForm.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  FormControl,
+  FormHelperText,
+  InputAdornment,
+  MenuItem,
+  Select,
+  TextField,
+} from "@material-ui/core";
+import EditOutlinedIcon from "@material-ui/icons/EditOutlined";
+import React from "react";
+
+import { IndustryOptions } from "@components/organisms/SetTargetGroup/constants";
+import JobAdStepContainer from "@pages/CreateJobAd/JobAdStepContainer";
+import JobPostingFieldContainer from "@pages/CreateJobAd/JobPostingFieldContainer";
+import JobStepNumber from "@pages/CreateJobAd/JobStepNumber";
+
+export interface JobCategory {
+  id: number;
+  name: string;
+  children?: Array<JobCategory>;
+}
+
+const JobPostingDetailsForm = ({
+  metadataValues,
+  jobFunctionList,
+  metaErrors,
+  setMetaFieldData,
+}: {
+  metadataValues: any;
+  jobFunctionList: Array<JobCategory>;
+  metaErrors: any;
+  setMetaFieldData: Function;
+}) => {
+  return (
+    <JobAdStepContainer>
+      <JobStepNumber number={1} title="Job posting details" />
+      {/* Job Title */}
+      <JobPostingFieldContainer
+        title="Job title"
+        description="This is the job title which will be shown to your candidates:"
```

Comment:
make all text literals in PR translatable.

Reply:
I tried to update all, should be good for now.